TheMovieDb/Modules/MovieDetail/MovieDetailLocalDataManager.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  MovieDetailLocalDataManager.swift
+//  TheMovieDb
+//
+//  Created by Javier Cueto on 14/11/21.
+//  
+//
+
+import Foundation
+
+class MovieDetailLocalDataManager: MovieDetailLocalDataManagerInputProtocol {
```

Comment:
if we are not using this. can we remove?

Reply:
Curiously, i had already deleted that file, because not appear y my project but in the finder it is 🧐